src/Dockerfile

```diff
@@ -3,6 +3,22 @@ LABEL maintainer="Jonas Alfredsson <jonas.alfredsson@protonmail.com>"
 
 ARG BUILDX_QEMU_ENV
 
+ENV CERTBOT_DNS_AUTHENTICATORS \
+    cloudflare \
+    cloudxns \
+    digitalocean \
+    dnsimple \
+    dnsmadeeasy \
+    gehirn \
+    google \
+    linode \
+    luadns \
+    nsone \
+    ovh \
+    rfc2136 \
```

Comment:
I actually saw that this plugin is configured the same as others, and thus someone could have their bind in a different container and be able to work with it. So I added it here too!

Reply:
Sounds good!